app/lib/tool/neat_task/pub_dev_tasks.dart

```diff
@@ -157,6 +157,8 @@ void _setupGenericPeriodicTasks() {
     ),
   );
 
+  _daily(name: 'topics', isRuntimeVersioned: false, task: countTopics);
```

Comment:
```suggestion
  _daily(name: 'count-topics', isRuntimeVersioned: false, task: countTopics);
```

Reply:
@isoos there is some try-catch sanity around this, right?

Reply:
Yeah, it will setup it in a sane way.

Reply:
done